code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -233,3 +233,156 @@
+
+	if(X.health < 0) //Gain additional rage with negative HP; gain + 0.0075 rage power per point of negative HP
+		rage_power += X.health * -0.0075
+
+	rage_power = min(1, rage_power) //Cap rage power so that we don't get way too insane.
+
+	var/rage_power_radius = CEILING(rage_power * 7, 1) //Define radius of the SFX
+
+	X.visible_message("<span class='danger'>\The [X] becomes frenzied, bellowing with a shuddering roar!</span>", \
+	"<span class='highdanger'>We bellow as our fury overtakes us! RIP AND TEAR!</span>")
+	X.do_jitter_animation(1000)
+
+
+	//Roar SFX; volume scales with rage
+	playsound(X.loc, 'sound/voice/alien_roar2.ogg', clamp(100 * rage_power, 25, 80), 0)
+
+	var/bonus_duration
+	if(rage_power > 0.5) //If we're super pissed it's time to get crazy
+		var/datum/action/xeno_action/charge = X.actions_by_path[/datum/action/xeno_action/activable/charge]
+		var/datum/action/xeno_action/ravage = X.actions_by_path[/datum/action/xeno_action/activable/ravage]
+		if(charge)
+			charge.clear_cooldown() //Reset charge cooldown
+		if(ravage)
+			ravage.clear_cooldown() //Reset ravage cooldown
+
+		for(var/mob/living/witness in hearers(rage_power_radius, X)) //Roar that applies cool SFX and some soft CC
+
+			if(!witness.hud_used)
+				continue
+
+			var/obj/screen/plane_master/floor/OT = witness.hud_used.plane_masters["[FLOOR_PLANE]"]
+			var/obj/screen/plane_master/game_world/GW = witness.hud_used.plane_masters["[GAME_PLANE]"]
+
+			addtimer(CALLBACK(OT, /atom.proc/remove_filter, "rage_outcry"), 1 SECONDS)
+			GW.add_filter("rage_outcry", 2, list("type" = "radial_blur", "size" = 0.07))
+			animate(GW.get_filter("rage_outcry"), size = 0.12, time = 5, loop = -1)
+			OT.add_filter("rage_outcry", 2, list("type" = "radial_blur", "size" = 0.07))
+			animate(OT.get_filter("rage_outcry"), size = 0.12, time = 5, loop = -1)
+			addtimer(CALLBACK(GW, /atom.proc/remove_filter, "rage_outcry"), 1 SECONDS)
+
+	for(var/turf/affected_tiles as() in RANGE_TURFS(rage_power_radius, X.loc))
+		affected_tiles.Shake(4, 4, 1 SECONDS) //SFX
+
+	for(var/mob/living/L in hearers(rage_power_radius, X)) //Roar that applies cool SFX and some soft CC
+		shake_camera(L, 1 SECONDS, 1)
+		L.Shake(4, 4, 1 SECONDS) //SFX
+
+		if(L.stat == DEAD) //We don't care about the dead
+			continue
+
+		if(isxeno(L))
+			var/mob/living/carbon/xenomorph/friendly_check = L
+			if(friendly_check.issamexenohive(X)) //No friendly fire
+				continue
+
+	X.add_filter("ravager_rage_outline", 5, outline_filter(1.5, COLOR_RED)) //Set our cool aura; also confirmation we have the buff
+
+
+	rage_plasma = min(X.xeno_caste.plasma_max - X.plasma_stored, X.xeno_caste.plasma_max * rage_power) //Calculate the plasma to restore (and take away later)
+	X.plasma_stored += rage_plasma //Regain a % of our maximum plasma scaling with rage
+
+	rage_sunder = min(X.sunder, rage_power * 100) //Set our temporary Sunder recovery
+	X.adjust_sunder(-1 * rage_sunder) //Restores up to 100 Sunder temporarily.
```

Comment:
the abilities does too much thing IMO.

plasma and damage ok, but sunder on top of it?

You're saying it's supposed to be a risky but rewarding ability, but the sunder makes it pretty good at avoiding risk as well

Reply:
If Sunder isn't temp replenished while the Ravager is in negatives it will easily melt before it has a chance to do anything.

Reply:
that means rage is a second get the fuck out mechanic, and it will be hard to die with a rav

Reply:
I mean if  you want to waste your minute cooldown Rage as a defensive tool, that's certainly a choice.